web/frontend/src/pages/election/components/utils/configurationSchema.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
Finally I found out a better way to implement the validation of the input configuration that can check everything in one place rather than having to define a JSON schema + do more JS manipulation to verify the coherence of the whole file. It's yup ([https://github.com/jquense/yup](url)) and it could also help for the validation in the web backend too, especially if we want to block any malicious user that would send broken data by short-cutting the front-end (i.e modified curl request).

Reply:
There is a major downfall with this solution: it is tight to javascript and will be less visible to the rest of the eco-system. The advantage of the JSON schema is that we can use it on the blockchain-backend as-well. It also serves as documenting the interface since it is not implementation-specific.